api/common/v1alpha1/ref.go

```diff
@@ -71,6 +71,17 @@ type TargetRef struct {
 	// all data plane types are targeted by the policy.
 	// +kubebuilder:validation:MinItems=1
 	ProxyTypes []TargetRefProxyType `json:"proxyTypes,omitempty"`
+	// Namespace specifies the namespace of target resource. If empty only resources in policy namespace
+	// will be targeted.
+	// Namespace can be used only with MeshService kind, and when targeting resources in Kubernetes zone.
```

Comment:
I would not mention the specific kind here. Technically `Namespace`, `Labels` and `SectionName` are not limited to `MeshService` and potentially can be used for referencing other kinds.

Reply:
changed